develop_docker/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.5'
```

Comment:
Do we still want this file?>

Reply:
I don't think this file is needed anymore. It should be easy enough to recreate it if needed

Reply:
still up for deletion?

Reply:
removed